portfolio/src/main/webapp/scripts.js

```diff
@@ -0,0 +1,18 @@
+/** Fetches the current date from the server and adds it to the page. */
+async function showServerString() {
+  const responseFromServer = await fetch('/MyServelt');
+  //const textFromResponse = await responseFromServer.text();
+  const stats = await responseFromServer.json();
+
+  //const dateContainer = document.getElementById('date-container');
+  //dateContainer.innerText = textFromResponse;
+  //const Quoat = stats;
+
+  // Pick a random greeting.
+  const Quoat = stats[Math.floor(Math.random() * stats.length)];
+
+  // Add it to the page.
```

Comment:
This comment is mostly unnecessary as it is very obvious what you're doing by the code itself. No need to update anything but before adding comments, always consider if the comment adds any information for someone reading the code.